### PR TITLE
Update remaining VEHICLE namespace hashed natives

### PR DESCRIPTION
**Pull request by ook3D**
added list of stations and lua example

### CODE REVIEW

**Comment by ook3D** [comment]
Didnt mean to commit the second changes to this branch, but ill explain the changes here anyways.

i went through and updated those that had already been solved and explained here: https://nativedb.dotindustries.dev/gta5/natives

of course not everything will be 100% correct and ready to go, just request changes if you spot any issues! thanks

**Comment by gottfriedleibniz** [comment]
> i went through and updated...

Wasting your time if you're not automating the process.

**Comment by ook3D** [comment]
> > i went through and updated...
> 
> Wasting your time if you're not automating the process.

i dont want to automate it because i wanted to ensure they were correct

File: VEHICLE/NetworkCapEmptyCrowdingVehiclesRemoval.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+ns: VEHICLE
+---
+## NETWORK_CAP_EMPTY_CROWDING_VEHICLES_REMOVAL
+
+```c
+// 0xA4A9A4C40E615885
+void NETWORK_CAP_EMPTY_CROWDING_VEHICLES_REMOVAL(int p0);
```
Comment (PsychoShock) [inline VEHICLE/NetworkCapEmptyCrowdingVehiclesRemoval.md:8]: Param is the threshold quantity of vehicles that, when surpassed, will initiate aggressive removal (if activated for this frame).

File: VEHICLE/SetCargobobPickupMagnetSetAmbientMode.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+ns: VEHICLE
+---
+## SET_CARGOBOB_PICKUP_MAGNET_SET_AMBIENT_MODE
+
+```c
+// 0x9BDDC73CC6A115D4 0x50CDB295
+void SET_CARGOBOB_PICKUP_MAGNET_SET_AMBIENT_MODE(Vehicle vehicle, BOOL p1, BOOL p2);
```
Comment (PsychoShock) [inline VEHICLE/SetCargobobPickupMagnetSetAmbientMode.md:8]: `p1` is if it affects the vehicle**s** and `p2` if it affects the object**s**.

File: VEHICLE/SetCheckForEnoughRoomForPed.md [hunk 3]
```diff
@@ -2,11 +2,11 @@
 ns: VEHICLE 
 ---
 
-## _0xEF9D388F8D377F44
+## SET_CHECK_FOR_ENOUGH_ROOM_FOR_PED
 
 ```c
 // 0xEF9D388F8D377F44 
-void _0xEF9D388F8D377F44(Vehicle vehicle, BOOL p1);
+void SET_CHECK_FOR_ENOUGH_ROOM_FOR_PED(Vehicle vehicle, BOOL p1);
```
Comment (PsychoShock) [inline VEHICLE/SetCheckForEnoughRoomForPed.md:9]: Second param is to check if it should check for enough room to fid the current ped.

**Review by PsychoShock — APPROVED** [review]
I review everything. Seems to be good. Missing a couple param's like I said. I'll maybe add more comments if @4mmonium wants to. Probably not a good idea to do BIG pr's like that since it takes a lot of times to review them in the future. Great PR tho! :D

**Pull request closed without merging**